include/aws/crt/mqtt/MqttClient.h

```diff
@@ -139,6 +139,14 @@ namespace Aws
                  */
                 bool SetWebsocketProxyOptions(const Http::HttpClientConnectionProxyOptions &proxyOptions) noexcept;
 
+                /**
+                 * Customize time to wait between reconnect attempts.
+                 * The time will start at min and multiply by 2 until max is reached.
+                 * The time resets back to min after a successful connection.
+                 * This function may only be called before Connect().
```

Comment:
trivial: It can also be called after the on_connection_completed callback, and it will be applied to the next reconnect process.

Reply:
For simplicity's sake, I'll change it to: "This function should be called before Connect()."

It's hard to imagine a world where someone wants to change these settings later